src/pubkey.h

```diff
@@ -205,4 +210,12 @@ struct CExtPubKey {
     bool Derive(CExtPubKey& out, unsigned int nChild) const;
 };
 
+/** Users of this module must hold an ECCVerifyHandle. */
```

Comment:
I've considered that, but it would be a mess to pass it down all the way from libconsensus' main function through all validation and script code to pubkey.cpp...